### PR TITLE
Issue 692 | Extend the form with the fields from the new specifications

### LINKED ISSUE #692: Extend the form with the fields from the new specifications
Extend the `new building` form to add the missing fields from the design

⚠️ Do not delete any of the existing fields, **even if they do not appear in the design**
ℹ️ All the new fields should be added between the **risk class input** and the **CAPTCHA verification**
🌐 The form must be internationalized, available in both English 🇬🇧  and Romanian 🇷🇴 

Specifications: https://docs.google.com/spreadsheets/d/1PbEHJWkgr2gDBz7ynqy4ia_po6cAbW_dcByRVl1fxks
Design: https://www.figma.com/file/dUL9ybnag2LhQpicJseFaH/Seismic-Risk?node-id=2630%3A3912
API schema: https://api-seismicrisc.code4.ro/api/v1/schema/swagger-ui/
Form endpoint: https://api-seismicrisc.code4.ro/api/v1/buildings/public_create/ (`OPTIONS` request)

### PR DESCRIPTION
**Pull request by vladplesu**
### What does it fix?

Closes #692 

### How has it been tested?

Manually

### CODE REVIEW

File: client/src/components/FormSection/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Col, Row, Typography } from 'antd';
+import React from 'react';
+import Chip from '../Chip';
+
+const { Title, Text } = Typography;
+
+const FormSection = ({ label, title, description, children }) => {
+  return (
+    <>
+      <Row type="flex" gutter={16} align="top" style={{ flexWrap: 'nowrap' }}>
```
Comment (cipick) [inline client/src/components/FormSection/index.js:10]: would be nice to have these inline styles extracted in css file

**Comment by cipick** [comment]
From QAing i noticed that this field is missing english translation: Screenshot 2022-01-06 at 17 39 36

Also this label is attached wrong to gdpr field
Screenshot 2022-01-06 at 17 39 07

File: client/package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
     "@lingui/react": "^3.13.0",
     "antd": "^3.26.12",
     "lodash": "^4.17.21",
-    "qs": "^6.10.2",
+    "prop-types": "^15.7.2",
+    "qs": "^6.10.1",
```
[the remark this answers is not in the document]
Comment (vladplesu) [reply client/package.json]: Done.

**Comment by vladplesu** [comment]
> From QAing i noticed that this field is missing english translation: Screenshot 2022-01-06 at 17 39 36
> 
> Also this label is attached wrong to gdpr field Screenshot 2022-01-06 at 17 39 07
> 
> But impressive work, nice job @vladplesu 👏

Thank you! I made the changes according to all your comments.

**Review by tudoramariei — APPROVED** [review]

**Pull request merged**